duckduckgo/duckduckgo-privacy-extension
# Rename allowlist to better reflect the wording of modern society

**Reviewer:** @kdzwinel 



## Description:

- In (https://app.asana.com/0/0/1200710125801716/f) the https list was migrated
  - Relevant e-tag code cleaned up on browser load.
- Migrated the old data to new.


## Steps to test this PR:

- Open Firefox
- Go to about:profiles, create new profile, enter "allowlist-migration" as it's name

1. Run "npm run dev-firefox"
2. Launch profile: web-ext run -s build/firefox/dev/ --p allowlist-migration
3. Go to DDG options page
4. "Add unprotected site"
5. Type "example.com" and click Add
6. Now visit "bbc.co.uk"
7. Click "Site Privacy Protection" toggle
8. Click "No"
9. Now visit "cnn.com"
10. Click "Site Privacy Protection" toggle
11. Click "Yes"
12. Send a report for "Something else"
13. Close all tabs besides "about:debugging"
14. Click "Load Temporary Add-on"
15. Checkout the migration branch
16. Check the settings panel lists all three sites
17. Check extension storage object has no legacy keys and they're now renamed to the new key names.
18. Visit example.com and check site toggle status



## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
